.circleci/config.yml

```diff
@@ -30,6 +30,7 @@ jobs:
               python -m venv venv || virtualenv venv
               . venv/bin/activate
               pip install -r dev-requirements.txt --quiet
+              pip install -e .
```

Comment:
Again, not a big deal but I'd think all `install -e` lines should go before all `install -r` (or any other versioned install) lines.